packages/test-utils/tsconfig.json

```diff
@@ -8,6 +8,9 @@
   "references": [
     {
       "path": "../replay"
+    },
+    {
+      "path": "../observability-node"
```

Comment:
we should also add an entry about this new pkg here:
https://github.com/replayio/replay-cli/blob/32788d3622377d8108dee4c7d7198d4a14d2b5e7/tsconfig.json

Reply:
ah, I'm a dummy - it's already there 🤦‍♂️